db/setup.rb

```diff
@@ -44,7 +44,8 @@
   start_date TEXT,
   end_date TEXT,
   slack_channel TEXT,
-  metadata TEXT
+  metadata TEXT,
+  active TEXT
```

Comment:
Would be useful to have a file to run that adds this new column and populates any existing projects that are in date as active (otherwise they will all be ignored).